operations/configuration_management/ARP_cache_tuning.md

```diff
@@ -0,0 +1,316 @@
+# ARP Cache Tuning Guide
+
+To ensure smooth communication on your network, ARP (Address Resolution Protocol) cache settings must be adjusted to handle a larger number of nodes. This guide will help you calculate and set the appropriate values for your system.
+
+## Prerequisites
+
+The following information is required and will need to be gathered before performing the steps outlined in this guide.
+
+- Total number of communicating nodes (master, worker, storage, compute and application nodes).
+- Total number of network interface cards in each node.
+
+---
+
+## Key ARP Cache Settings
+
+1. **`gc_thresh1`**: Minimum number of entries the cache attempts to maintain.
+2. **`gc_thresh2`**: Threshold where older entries start getting cleared before the cache overfills.
+3. **`gc_thresh3`**: Maximum number of cache entries allowed. If exceeded, new entries may be dropped.
+4. **`gc_stale_time`**: Time (in seconds) before an ARP entry is marked stale.
+5. **`base_reachable_time_ms`**: Time (in milliseconds) an entry remains valid before being re-verified.
+
+---
+
+## Formula for Tuning ARP cache settings
+
+Storage nodes do not have HSN connections. To keep arp settings management simple, the values calculated for the worker nodes should also be applied to the storage nodes.
+
+To calculate the base `gc_thresh1` value:
+
+**`gc_thresh1`**:
+
+```text
+gc_thresh1 = (number of nodes) × ((NICS per node)²)
+```
+
+Where:
+
+- **`number of nodes`** = Total number of communicating nodes (master, worker, storage, compute and application nodes).
+- **`NICS per node`** = Number of NICs in each node (HSN NICs + NMN NICs). Nodes will have 2, 3, or 5 total NICs.
+    - HSN NICs = number of NICs on the HSN per node. If some compute nodes have 1, some have 2, and some have 4 HSN NICs, then use the largest value of **`4`** HSN NICs per node.
+    - NMN NICs = number of NICs on the NMN per node. Most nodes have a single NIC on the NMN, but workers will have two. For the formula, assume each node has **`1`** NIC On the NMN.
+
+### Calculate Remaining Values
+
+- `gc_thresh2 = 1.5 × gc_thresh1`
+- `gc_thresh3 = 2 × gc_thresh2`
+
+---
+
+## Example Calculation
+
+**Scenario**:
+
+- Total nodes = 1000 (master, worker, storage, compute and application nodes).
+- Each node has:
+    - 4 HSN NICs.
+    - 1 NMN NIC.
+    - **NICS per node = 4 + 1 = 5**
+
+**`gc_thresh1`**:
+
+```text
+gc_thresh1 = 1000 × (5²) = 25000
+```
+
+**`gc_thresh2`**:
+
+```text
+gc_thresh2 = 1.5 × 25000 = 37500
+```
+
+**`gc_thresh3`**:
+
+```text
+gc_thresh3 = 2 × 37500 = 75000
+```
+
+**Example settings**:
+
+```bash
```

Comment:
I believe `ini` should work, otherwise use `text` to prevent bash highlighting weird things
```suggestion
```ini
```